server/adaptors/integrations/__data__/repository/aws_elb/schemas/aws_elb-1.0.0.mapping.json

```diff
@@ -0,0 +1,98 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+        "version": "1.0.0",
+        "catalog": "observability",
+        "type": "logs",
+        "component": "aws_elb"
+      },
+      "properties": {
+        "aws": {
+          "type": "object",
+          "properties": {
+            "elb": {
+              "properties": {
+                "backend.ip": {
+                  "type": "ip"
+                },
+                "backend.port": {
+                  "type": "integer"
+                },
+                "backend.processing_time": {
+                  "type": "half_float"
+                },
+                "backend.status_code": {
+                  "type": "short"
+                },
+                "client.ip": {
+                  "type": "ip"
+                },
+                "client.port": {
+                  "type": "integer"
+                },
+                "connection_time": {
+                  "type": "integer"
+                },
+                "destination.ip": {
+                  "type": "ip"
+                },
+                "destination.port": {
+                  "type": "integer"
+                },
+                "elb_status_code": {
+                  "type": "short"
+                },
+                "http.port": {
+                  "type": "integer"
+                },
+                "http.version": {
+                  "type": "keyword"
+                },
```

Comment:
Still have flat fields with multiple components instead of just encoding nested objects, does the inconsistency cause any problems for opensearch? If they're equivalent then it's probably fine.